pkg/provisioner/controller.go

```diff
@@ -261,12 +256,15 @@ func (c *obcController) syncHandler(key string) error {
 
 // handleProvision is an extraction of the core provisioning process in order to defer clean up
 // on a provisioning failure
-func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBucketClaim, class *storagev1.StorageClass) (err error) {
+func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBucketClaim, class *storagev1.StorageClass) error {
+
+	log.Info("syncing obc creation")
 
 	var (
 		ob        *v1alpha1.ObjectBucket
 		secret    *corev1.Secret
 		configMap *corev1.ConfigMap
+		err       error
```

Comment:
Is there value defining `err` separately in the func body vs as a named return?

Reply:
I don't think it's necessarily idiomatic but my sense is that it hurts readability in large functions.  Particularly because we already have a block defining local variables.

Reply:
Ack. Especially:
> Particularly because we already have a block defining local variables.